libass/ass_fontselect.c

```diff
@@ -185,6 +190,80 @@ static void load_fonts_from_dir(ASS_Library *library, const char *dir)
         }
     }
     closedir(d);
+#else
+    // The dir name must ending with \*, D:\temp not work, D:\temp\* works well.
+    size_t dir_len = strlen(dir);
+    char* tdir = malloc(dir_len + 3);
+    strcpy(tdir, dir);
+    strcat(tdir, "\\*");
```

Comment:
Will this also work when the path already ends in either a `/` or `\`?

Reply:
No. [MS document](https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-findfirstfilew#parameters) say:
> This parameter should not be NULL, an invalid string (for example, an empty string or a string that is missing the terminating null character), or end in a trailing backslash (\).

Reply:
I am sorry I misunderstand it.
But it works:
![image](https://user-images.githubusercontent.com/41434272/124336053-21feec00-dbcf-11eb-9696-e9b70df5dcd6.png)